src/views/Main/Catalog/Components/EdmDatatypeAutosuggest.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import Autosuggest from 'react-autosuggest';
+import Utils from '../../../../utils/Utils';
+import '../../../../styles/autosuggest.css';
+
+const getSuggestionValue = suggestion => suggestion;
+
+const renderSuggestion = suggestion => (
+  <div>
+    {suggestion}
+  </div>
+);
+
+export class EdmDatatypeAutosuggest extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      value: '',
+      suggestions: []
+    };
+  }
+
+  getSuggestions(value) {
+    const inputValue = value.trim().toLowerCase();
```

Comment:
the list of datatypes is not upper case, but I could just manually change the list to avoid having to do a second toLowerCase().